docs/index.md

```diff
@@ -130,7 +133,7 @@ On variants of Linux you may be prompted for additional package installation to
 
   (Note the prompt `Installing to the system store is not yet supported on this Linux`, which can be a simple result of not having /usr/sbin in the path so that `/usr/sbin/update-ca-certificates` can be found.)
 
-### Windows and Firefox `mkcert -install` additional instructions
+### Windows/Firefox additional instructions
```

Comment:
Are these instructions limited to just Windows?

Reply:
Yes, only a problem on Windows. On macOS and linux Firefox uses the system CA.